resources/views/users/dashboard/membercontract.blade.php

```diff
@@ -0,0 +1,117 @@
+@extends('website.layouts.redesign.generic')
+
+@section('page-title')
+    Sign membership contract
+@endsection
+
+
+@section('javascript')
+    @parent
+    <script>
+        let signatureAlert = $('#signature-alert');
+        let signatureForm = $('#signature-form');
+        let signatureField = $('#signature');
+        let signatureDrawn = false
+
+        signatureAlert.hide();
+
+        signatureField.jqSignature({
+            width: 350,
+            height: 300,
+            lineWidth: 3
+        });
+
+        signatureField.on('jq.signature.changed', function() {
+            signatureDrawn = true;
+            signatureAlert.hide();
+            $("input[name='signature']").val(signatureField.jqSignature('getDataURL'));
+        });
+
+        function clearCanvas() {
+            signatureField.jqSignature('clearCanvas');
+            signatureDrawn = false;
+        }
+
+        signatureForm.submit(function(e) {
+            if (!signatureDrawn) {
+                e.preventDefault();
+                signatureAlert.show();
+            }
+        })
+
+    </script>
+@endsection
+
+@section('container')
+
+    <div class="row justify-content-center">
+
+        <div class="col-lg-6 col-md-7" style="max-width: 600px">
+
+                <div class="card mb-3">
+
+                    <div class="card-header bg-dark text-white">@yield('page-title')</div>
+
+                    <div class="card-body">
+                        <p>
```

Comment:
We now have this text on two places. We should either merge this view with `resources/views/users/admin/membershipform.blade.php` or store the HTML content in a third view and include it in both views.